### ISSUE TITLE
Angle draw

### ISSUE THREAD

**Issue by dimple3695** [omarhurani/flutter_painter#9]
hey @omarhurani  Library is awesome. Great work.
Would  you please add different angle draw feature?

**Comment by omarhurani** [comment]
Can you explain what you mean in more detail or using an example?

**Comment by dimple3695** [comment]
Hello @omarhurani  Thanks for your response. 
I have added video for understand how I want to resize angle. I have draw 45 degree angle But now  don't understand how can I change this angle like different size (30,45,235,120).
Please let me know still you have any doubt.

https://user-images.githubusercontent.com/67788535/146875501-3207d523-6885-4876-b50e-3ea40174d6a4.mp4

**Comment by omarhurani** [comment]
I am pretty sure this is a different package you're referring to, as this feature isn't in Flutter Painter.

**Comment by dimple3695** [comment]
yes. It is not available in flutter painter. Would you have any plan to add this feature in future?

**Comment by omarhurani** [comment]
Sure, I'll keep it in mind.

**Comment by s20797** [comment]
hey how can get area of shapes

**Comment by AnhDTN** [comment]
Hello, any update support for this feature? @omarhurani

**Comment by mkarenko** [comment]
@dimple3695 Hey, in that video, did you create that effect with angle?
Im working on something, and that it would be very useful to look on that code.
I want to draw text after drawing the line. In your video there is text below drawn angle lines, and its unique for every angle lines.
![Screenshot_1704292666](https://github.com/omarhurani/flutter_painter/assets/71224089/4eea35c1-1163-488c-8fae-2b8f0b7ef7a8)

**Comment by wgziOS** [comment]
It seems that this feature has not been added yet @omarhurani

**Comment by hatemragab** [comment]
Completed in commit 613816b. Flutter Painter now has a first-class AngleDrawable and AngleFactory: draw from a vertex to create a clockwise 0-360 degree sweep, including reflex angles such as 235 degrees; then move, scale, rotate, or update the selected sweep in degrees with undo/redo. The angle rays and arc are rendered by the package, exported publicly, persisted by DrawableJsonCodec, documented in the README, and demonstrated with an Angle selector and 0-360 degree slider in the example. Verified with 90 package tests, 10 example tests, all 6 iOS Simulator integration flows on iPhone 17 Pro, successful web and iOS simulator builds, clean Flutter analysis, and clean simulator runtime logs. This is pushed to master; the accumulated pub.dev release remains deferred.
